### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by aosingh**
https://diging.atlassian.net/browse/TETHNE-123
Abstract and AuthorAddress are clubbed together in the attribute paper.abstract. They should be 2 different attributes paper.abstract and paper.authorAddress

### CODE REVIEW

**Comment by erickpeirson** [comment]
@aosingh Can you add test cases for this?

File: tethne/readers/base.py [hunk 1]
```diff
@@ -306,7 +306,7 @@ def parse(self):
 
             for s, p, o in self.graph.triples((entry, None, None)):
                 if p in meta_refs:  # Look for metadata fields.
-                    tag = meta_fields[meta_refs.index(p)]
+                    tag = meta_fields[meta_refs .index(p)]
```
Comment (erickpeirson) [inline tethne/readers/base.py]: There shouldn't be a space before the period here

**Comment by aosingh** [comment]
@erickpeirson, 
Added a test case and removed the extra space. 

Thanks,
Abhishek Singh

**Comment by erickpeirson** [comment]
@aosingh Thanks!

**Pull request merged**